server/cluster/cluster.go

```diff
@@ -1207,6 +1207,9 @@ func (c *RaftCluster) DeleteStoreLabel(storeID uint64, labelKey string) error {
 		return errs.ErrInvalidStoreID.FastGenByArgs(storeID)
 	}
 	newStore := typeutil.DeepClone(store.GetMeta(), core.StoreFactory)
+	if len(newStore.GetLabels()) == 0 {
+		return errors.Errorf("the label key %s does not exist", labelKey)
+	}
```

Comment:
Should we check it before cloning?

Reply:
Clone store before L1223(newStore.Labels = labels)?